### ISSUE TITLE
For `navigation list item`, only external navigation should be a link

### ISSUE THREAD

**Issue by pya35** [Orange-OpenSource/ouds-ios#1718]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-ios/issues) for duplicate or closed feature requests

### Your test device

iPhone 17

### Device OS version

iOS26

### Is Liquid Glass disabled

- [ ] Yes, Liquid Glass is disabled (or OS version is older than 26)
- [ ] No, Liquid Glass is enabled (or OS version is 26 or newer)

### OUDS iOS library version

3.0.0

### Describe the issue

All types of navigation have a link role: next, previous or external.

### Expected Behavior

If the component is used with **next** or **previous** indicator, then it should have a button role.
If the component is used with **external** indicator, then it should have a link role.

### Device settings

Voice Over

**Comment by pylapp** [comment]
📣 **New TestFlight BETA upload available** 🚀

Please, get the new build from *TestFlight* app, or contact the project maintainers (pylapp, ludovic35) to get access and test this issue implementation.

- Display name: **Design System Toolbox (BETA)**
- Version: **3.0.0**
- Build number: **1788815142**
- Build tag: **d979f56**


_🤖 This is an automated message sent with love thanks to *Fastlane* from our CI/CD pipeline 🤘_

**Comment by pylapp** [comment]
📣 **New TestFlight BETA upload available** 🚀

Please, get the new build from *TestFlight* app, or contact the project maintainers (pylapp, ludovic35) to get access and test this issue implementation.

- Display name: **Design System Toolbox (BETA)**
- Version: **3.0.0**
- Build number: **1788989831**
- Build tag: **5e2d2a8**


_🤖 This is an automated message sent with love thanks to *Fastlane* from our CI/CD pipeline 🤘_

**Comment by pylapp** [comment]
📣 **New TestFlight BETA upload available** 🚀

Please, get the new build from *TestFlight* app, or contact the project maintainers (pylapp, ludovic35) to get access and test this issue implementation.

- Display name: **Design System Toolbox (BETA)**
- Version: **3.0.0**
- Build number: **1789074302**
- Build tag: **bb28c6d**


_🤖 This is an automated message sent with love thanks to *Fastlane* from our CI/CD pipeline 🤘_

**Comment by pylapp** [comment]
📣 **New release available** 🚀

Please, get the [new release from GitHub](https://github.com/Orange-OpenSource/ouds-ios-design-system-toolbox/releases/tag/3.0.0) and wait for App Store update to test it!

- Display name: **Design System Toolbox**
- Version: **3.0.0**
- Build number: **1789078720**



_🤖 This is an automated message sent with love thanks to *Fastlane* from our CI/CD pipeline 🤘_
